edb/pgsql/codegen.py

```diff
@@ -113,33 +114,41 @@ def generate_ctes_source(
 
 
 class TranslationData:
```

Comment:
Can we rename this to something less generic? "Translation data" could be anything that compiler produces. I suggest "SourceMap".

Reply:
I'll do a separate clean PR for that rename